Testing/ORKTest/ORKTest/MainViewController.m

```diff
@@ -3090,12 +3090,12 @@ - (UIImage *)imageWithColor:(UIColor *)color size:(CGSize)size border:(BOOL)bord
 /*
  Dismisses the task view controller.
  */
-- (void)dismissTaskViewController:(ORKTaskViewController *)taskViewController {
+- (void)dismissTaskViewController:(ORKTaskViewController *)taskViewController removeOutputDirectory:(BOOL)removeDir{
```

Comment:
Add space before `{`.

Also I'd personally use the full `removeOutputDirectory` name for the argument variable.